instrumentation/action_mailer/README.md

```diff
@@ -67,9 +67,9 @@ end
 
 ## Semantic Conventions
 
-Internal spans are named using the name of the `ActiveSupport` event that was provided (e.g. `action_mailer deliver`).
+Internal spans are named using the name of the `ActiveSupport` event that was provided (e.g. `deliver.action_mailer`).
 
-The following attributes from the notification payload for the `deliver.action_mailer` event are attached to `action_mailer deliver` spans:
+The following attributes from the notification payload for the `deliver.action_mailer` event are attached:
```

Comment:
Could you add a section describing the attributes captured by the `process.action_mailer` notification by default? 
https://edgeguides.rubyonrails.org/active_support_instrumentation.html#process-action-mailer